src/claude/utils/sessionScanner.ts

```diff
@@ -21,51 +32,75 @@ export async function createSessionScanner(opts: {
     let currentSessionId: string | null = null;
     let watchers = new Map<string, (() => void)>();
     let processedMessageKeys = new Set<string>();
+    // Track skipped message IDs to avoid log spam
+    let skippedMessageIds = new Set<string>();
```

Comment:
Can't we just add these to processedMessageKeys? Skipped is technically also processed.